build.gradle

```
@@ -1,6 +1,8 @@
 plugins {
     id "com.jfrog.bintray" version "1.1"
     id "com.github.jruby-gradle.base" version "1.5.0"
+    id "com.palantir.git-version" version "0.12.3"
+    id "com.diffplug.spotless" version "6.12.1"
```

`6.13.0` is  the last version available for Java 8
refs: https://github.com/diffplug/spotless/blob/gradle/6.14.0/plugin-gradle/CHANGES.md#6140---2023-01-26
```suggestion
    id "com.diffplug.spotless" version "6.13.0"
```